include/aspect/stokes_matrix_free.h

```diff
@@ -224,6 +261,18 @@ namespace aspect
         void fill_cell_data (const Table<2, VectorizedArray<number>> &viscosity_table,
                              const bool is_compressible);
 
+        /**
+         * Fills in the tables of derivatives and strain rate for Newton method,
+         * and gives information regarding stabilization schemes.
```

Comment:
I am not sure what you mean by *gives* information. The function does not return anything. Do you mean it *computes* this information and stores it internally somewhere for later retrieval? Or does it only compute this information and uses it and forgets about it afterwards? In this case you don't need to mention it here.

I realize this is similar wording as for the function above, but there I do not understand what it means either :-).